src/client/components/general/IngredientCard/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './ingredientCard.scss'
+
+function index({ name, img, description, index }) {
```

Comment:
Change this line to:
`export default function IngredientCard({ name, img, description, index }) {`

With export default before the function, you can remove line 15 from this component